server/resources/translations/common.en.json

```diff
@@ -4,7 +4,7 @@
   "alertCompleted": "Thank you! This session is now complete. (You don't need to respond to this message.)",
   "alertFallback": "An alert to check on {{ deviceDisplayName }} was not responded to. Please check on it.",
   "alertReminder": "This is a reminder to check on {{ deviceDisplayName }}",
-  "alertStart": "This is a {{ alertTypeDisplayName }} alert. Please check on {{ deviceDisplayName }}. Please respond with 'ok' once you have checked on it.",
+  "alertStart": "This is a {{ alertTypeDisplayName }} alert. Please check on {{ deviceDisplayName }}. Please respond with 'yes' once you have checked on it.",
```

Comment:
Thanks. But I actually preferred 'ok', can we change this back?

Reply:
'ok'
yep no problem i will do that : )